recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   run:
     - python >=3
     - geopandas
-    - matplotlib >=2
+    - matplotlib-base >=2
```

Comment:
@nkorinek - i think we can potentially update our earthpy dependencies to  only rquire matplotlib base.  do  we  need  QT for anything?

Reply:
@lwasser I don't believe so...we can update it and see if anything breaks!